src/stan-users-guide/README.md

```diff
@@ -1,111 +1,19 @@
-# The Stan Book
+Steps to build functions manual:
 
-This is the repository for *Bayesian Statistics Using Stan", which
-serves as both the Stan users' guide and an introduction to Bayesian
-statistics.
+1.  Install R packages:  bookdown
 
-### Directory Structure
+2.  Install on OS and ensure on PATH: pandoc, pandoc-citeproc,
+pdflatex
 
-* `*.Rmd` files: basic text
-* `_bookdown.yml`: book includes
-* `_output.yml`: output config
-* `bib/all.bib`: BibTeX file for references
+3.  To build single manual, such as cmdstan-guide, functions-reference, reference-manual, or stan-users-guide, using Rscript
+    a. In shell change directory to `docs/src/<MANUAL>`
+    b. In shell, execute 
+       ```
+       # build html
+       > Rscript -e "bookdown::render_book('index.Rmd', output_format='bookdown::gitbook')"
+       # build pdf
+       > Rscript -e "bookdown::render_book('index.Rmd', output_format='bookdown::pdf_book')"
+       ```
+       The output will be written to subdirectory `_build`.
 
```

Comment:
why are you removing this information?

Reply:
Because it is outdated and confusing to a user like me.